troposphere/static/js/components/admin/ImageRequest.react.js

```diff
@@ -136,141 +136,143 @@ export default React.createClass({
 
         return (
         <div className="request-admin pull-right admin-detail">
-            <div>
-                <h3>Request Information</h3>
+            <div style={{ marginBottom: "20px" }}>
+                <h3 className="t-body-2">Request Information</h3>
                 <div>
-                    Request ID:
-                    {request.get("id")}
+                   Request ID:
+                   {` ${request.get("id")}` }
```

Comment:
Yeah, I'm doing this to maintain the space after `Request ID:`. 
An alt might be:
```
{ "Request ID: " + request.get("id") }
```